pr-data.csv

```diff
@@ -5243,7 +5243,7 @@ https://github.com/orbit/orbit,c4904af2f3745fea93871493edd7e09348e77768,actors/r
 https://github.com/orbit/orbit,c4904af2f3745fea93871493edd7e09348e77768,actors/test/actor-tests,cloud.orbit.actors.test.GenericActorStateTest.testGenericStateActor,ID,Deleted,,
 https://github.com/orbit/orbit,c4904af2f3745fea93871493edd7e09348e77768,actors/test/actor-tests,cloud.orbit.actors.test.OnlyIfActivatedTest.onlyIfActivatedTest,NIO,Deleted,,
 https://github.com/orbit/orbit,c4904af2f3745fea93871493edd7e09348e77768,actors/test/actor-tests,cloud.orbit.actors.test.serialization.JsonReferenceSerializationTest.testClassWithInnerValues,ID,Deleted,,
-https://github.com/orika-mapper/orika,27dd4c4666f2c559c0ef84d18221cf104b1f699c,core,ma.glasnost.orika.test.generator.BeanToArrayGenerationTestCase.testBeanToArrayGeneration,ID,Opened,https://github.com/orika-mapper/orika/pull/403,
+https://github.com/orika-mapper/orika,27dd4c4666f2c559c0ef84d18221cf104b1f699c,core,ma.glasnost.orika.test.generator.BeanToArrayGenerationTestCase.testBeanToArrayGeneration,ID,,,
```

Comment:
Why are you undoing this change? Can you rebase your work with your `main` synced to the original repo?

Reply:
Not all. Two of them are assigned to me. One is my teammate's.

Reply:
Sorry, I forgot to do that. What should I do to resolve this issue?

Reply:
Sync your `main` branch (in your forked repo) with `main` in the original repo (TestingResearchIllinois/idoft).

Get your `flaky-move-DataFlowTemplates` branch to have everything from `main` and then only one commit with your change(s).

Reply:
Done. Sorry for the inconvenience.